pkg/storage/fs/owncloud/owncloud.go

```diff
@@ -2155,16 +2155,17 @@ func (fs *ocfs) RestoreRecycleItem(ctx context.Context, key string) error {
 		return nil
 	}
 
-	origin := "/"
-	if v, err := xattr.Get(src, trashOriginPrefix); err != nil {
-		log.Error().Err(err).Str("key", key).Str("path", src).Msg("could not read origin")
-	} else {
-		origin = filepath.Clean(string(v))
+	if restorePath == "" {
+		v, err := xattr.Get(src, trashOriginPrefix)
+		if err != nil {
+			log.Error().Err(err).Str("key", key).Str("path", src).Msg("could not read origin")
```

Comment:
Shouldn't we return here? Because if `err != nil`, `restorePath` will be the base of src, restoring to which might create conflicts

Reply:
It also restored to `/` before, I didn't want to change behaviour like that with this PR.
Having said that I think the origin xattr will only be missing in very bad exceptions anyway. In those cases it might be desirable to have the files restored to `/` (if possible), no?